package.json

```diff
@@ -140,11 +140,12 @@
     "istanbul": "latest",
     "mocha": "latest",
     "nock": "2.17.0",
+    "nodemon": "^1.11.0",
```

Comment:
We could possibly use `latest` here - as far as I understand, since `nodemon` doesn't really interfere with our code, it shouldn't lead to incompatibilities.